### PR TITLE
[WIP] Integrate Socorro Lens with bmo

### PR DESCRIPTION
**Pull request by SebastinSanty**
@dylanwh @ashughes1 : Can you review this?

### CODE REVIEW

File: socorro-lens.html [hunk 1]
```diff
@@ -0,0 +1,148 @@
+<html lang="en">
+<head>
+    <meta content="text/html; charset=utf-8" http-equiv="Content-Type">
+    <link href='https://fonts.googleapis.com/css?family=Open+Sans:400,300,700' rel='stylesheet' type='text/css'>
+    <link href='https://fonts.googleapis.com/css?family=PT+Serif:400,700,400italic' rel='stylesheet' type='text/css'>
+    <link href='https://netdna.bootstrapcdn.com/font-awesome/4.2.0/css/font-awesome.css' rel='stylesheet' type='text/css'>
+    <link href='https://maxcdn.bootstrapcdn.com/bootstrap/3.2.0/css/bootstrap.min.css' rel='stylesheet' type='text/css'>
+    <link href="js/metricsgraphics/metricsgraphics.css" rel="stylesheet" type="text/css" id="dark">
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.1/jquery.min.js"></script>
```
Comment (dylanwh) [inline socorro-lens.html]: use the version of jquery the ships with BMO

File: socorro-lens.html [hunk 2]
```diff
@@ -0,0 +1,148 @@
+<html lang="en">
+<head>
+    <meta content="text/html; charset=utf-8" http-equiv="Content-Type">
+    <link href='https://fonts.googleapis.com/css?family=Open+Sans:400,300,700' rel='stylesheet' type='text/css'>
```
Comment (dylanwh) [inline socorro-lens.html]: include this css with BMO (and the subsequent ones too)

File: socorro-lens.html [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+    <link href='https://fonts.googleapis.com/css?family=PT+Serif:400,700,400italic' rel='stylesheet' type='text/css'>
+    <link href='https://netdna.bootstrapcdn.com/font-awesome/4.2.0/css/font-awesome.css' rel='stylesheet' type='text/css'>
+    <link href='https://maxcdn.bootstrapcdn.com/bootstrap/3.2.0/css/bootstrap.min.css' rel='stylesheet' type='text/css'>
+    <link href="js/metricsgraphics/metricsgraphics.css" rel="stylesheet" type="text/css" id="dark">
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.1/jquery.min.js"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/d3/3.4.11/d3.min.js" charset="utf-8"></script>
+    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.2.0/js/bootstrap.min.js"></script>
```
Comment (dylanwh) [inline socorro-lens.html]: both bootstrap and d3 would need to be included.

File: socorro-lens.html (hunk 1 above)
Comment (SebastinSanty) [inline socorro-lens.html]: Yeah, I'll do that

File: socorro-lens.html (hunk 3 above)
Comment (SebastinSanty) [inline socorro-lens.html]: I have included them, and tried working with them, no errors in inspector, still not showing up. Only CDN's were working for me. Can you check from your side, what might be the issue?

File: socorro-lens.html (hunk 2 above)
Comment (SebastinSanty) [inline socorro-lens.html]: Sure, I'll do that.

File: socorro-lens.html (hunk 3 above)
Comment (floatingatoll) [inline socorro-lens.html]: I can help if you're still having trouble on that, but will need a rebased patch to be of more use.

**Pull request closed without merging**